SPECS/python3/python3.spec

```diff
@@ -1,34 +1,22 @@
 %global openssl_flags -DOPENSSL_NO_SSL3 -DOPENSSL_NO_SSL2 -DOPENSSL_NO_COMP
 %global __brp_python_bytecompile %{nil}
-# Automating the extraction of these alternate version strings has proven to be tricky,
-# with regards to tooling available in the toolchain build environment.
-# These will be manually maintained for the time being.
-%global majmin 3.9
-%global majmin_nodots 39
-# See Lib/ensurepip/__init__.py in Source0 for these version numbers
-%global pip_version 22.0.4
-%global setuptools_version 58.1.0
+%define majmin %(echo %{version} | cut -d. -f1-2)
```

Comment:
Clarification: I recall `rpm` tools used to break when we used macros before they were defined; `version` in this case.

Reply:
thanks for the heads up. but this appears to be fixed with rpm 4.18.1 - I've confirmed these are defined properly